sa/model.go

```diff
@@ -216,6 +263,33 @@ type regModel struct {
 	Status    string `db:"status"`
 }
 
+// contactsToJSON converts a slice of contact addresses to a JSON array.
```

Comment:
It's not clear to me why these contactsToJSON and jsonToContacts functions got broken out, when they're still only called in one place (where they used to be in-lined). Seems like maybe because you thoughts you were going to call them directly, but ended up going through registrationModelToPb and registrationPbToModel instead?

Since they still only have one callsite, I wouldn't break these out yet.

Reply:
Good catch, and your intuition is correct! I had some code to explicitly call them but I wound up not using it. I think the broken-out functions are very slightly neater but they're not relevant to this change so I'm reverting.